com.wps.WPS.appdata.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.wps.WPS</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=http://wps-community.org/license.md</project_license>
+  <name>WPS Office</name>
+  <summary>WPS Office Suite</summary>
+  <description>
+    <p>WPS Office including Writer, Presentation
+    and Spreadsheets, is a powerful office suite, which is able to process
+    word file, produce wonderful slides, and analyze data as well. It is
+    deeply compatible with all of the latest Microsoft Office file formats.
+    It can easily open and read the documents created with Microsoft Office.</p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or
+    supported by Kingsoft Office Corporation</p>
+  </description>
+  <launchable type="desktop-id">com.wps.WPS-wps.desktop</launchable>
```

Comment:
This "sub" app-ID should actually be com.wps.WPS.* rather than WPS-foo. Strictly speaking, com.wps.WPS-foo is a different top-level app-ID to com.wps.WPS. Flatpak allows the -foo suffix for .desktop files (which is arguably a bug), but not for appdata, so using - rather than . means it's not possible to ship appdata for the separate components or associate them back to the Flatpak, so they can't be shown/launched in GNOME Software for example.

Reply:
Also the .desktop suffix is a little ambiguous here and no longer required/recommended.

Reply:
I refered to the Libreoffice appdata. The components are all separated by '-'.